power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/math_solver.hpp

```diff
@@ -11,16 +11,13 @@
 #include "../power_grid_model.hpp"
 #include "../three_phase_tensor.hpp"
 #include "../timer.hpp"
```

Comment:
maybe create a separate regex for this type of entries. they lie outside the current "submodule" and therefore should be treated as external relative to the current module. And yes that may cause some files to first include local things and then `../power_grid_model.hpp`, but in those cases, that `../power_grid_model.hpp` index may be removed anyway, so there is no issue